src/i18n/strings.ftl

```diff
@@ -55,5 +60,16 @@ SIMILAR-SPECIES-header
 # Header for a block of text describing a taxon's conservation status
 STATUS-header = STATUS
 
+Take-a-photo-with-your-camera = Take a photo with your camera
+
 # Header for a block of text describing a taxon's taxonomy
 TAXONOMY-header = TAXONOMY
+
+Upload-a-photo-from-your-gallery = Upload a photo from your gallery
+
+Upload-X-photos = Upload {$count ->
+    [one] 1 photo
+    *[other] {$count} photos
+}
+
+You-can = You can:
```

Comment:
This could use a context comment. Someone is eventually going to ask "You can what?" b/c either the pronoun or the verb will be inflected differently depending on what follows.